frontend/src/DataImporter/parseCsv.ts

```diff
@@ -77,7 +77,7 @@ const remapChannelWidth = (frequency: number): number => {
 export const parseEdgeSpectrum = (data: string): EdgeSpectrumDataRow[] => {
   const lines = parseCsv(data);
   return lines.map(([edgeId, channelId, frequency, channelWidth, _wavelength, _channelLabel]) => {
-    if (edgeId === '' || channelId === '' || frequency === '' || channelWidth === '') {
+    if (edgeId === '' || channelId === '' || frequency === '' || channelWidth === '' || _channelLabel === '') {
```

Comment:
imo to jest źle bo ustaliliśmy że my się opieramy na photonic service id czyli tutaj polu channelId, a channelLabel nie wszędzie jest uzupełnione więc je ignorujemy